flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriter.java

```diff
@@ -170,30 +170,30 @@
      * the valid limits of the destination). The logic then needs to create and execute the request
      * asynchronously against the destination (ideally by batching together multiple request entries
      * to increase efficiency). The logic also needs to identify individual request entries that
-     * were not persisted successfully and resubmit them using the {@code requestResult} callback.
+     * were not persisted successfully and resubmit them using the {@code requestToRetry} callback.
```

Comment:
I understand the Jira said `requestToRetry`, but given this is a `List<>` I think a better name would be `requestsToRetry`, what do you think?

Reply:
For example, it is used to call the method of accept() ->  requestToRetry.accept(retryableFailedRecords)
@dannycranmer  If you still think requestsToRetry is better, I will change to  requestsToRetry

Reply:
ok makes sense